### PR TITLE
Fixes silicons immunities to ash and rad storms

### PR DESCRIPTION
**Pull request by Cyberboss**
Why is it only 10AM reeeeee

### CODE REVIEW

File: code/modules/mob/living/silicon/silicon.dm [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	verb_yell = "alarms"
 	see_in_dark = 8
 	bubble_icon = "machine"
-	weather_immunities = list("ash")
+	weather_immunities = list("ash", "rad")
```
Comment (RemieRichards) [inline code/modules/mob/living/silicon/silicon.dm]: ical

Comment (Cyberboss) [inline code/modules/mob/living/silicon/silicon.dm]: ???

Comment (GunHog) [inline code/modules/mob/living/silicon/silicon.dm]: It took me about 6 seconds too, Cyberboss, do not be sad!

**Comment by Cyberboss** [comment]
Actually i was hoping to fix them getting the "you feel waves of heat wash over you" message but immunities don't do that. And once #21478 gets merged this won't matter

**Pull request closed without merging**